lib-android/src/main/java/com/devinci/lib/view/ViewUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.devinci.lib.view;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.content.res.Resources;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import android.support.annotation.ColorRes;
+import android.support.annotation.DrawableRes;
+import com.devinci.lib.util.Versions;
+
+public enum ViewUtils {
+  ;
+
+  @TargetApi(Build.VERSION_CODES.LOLLIPOP) @SuppressWarnings("deprecation")
+  public static Drawable getDrawable(Context context, @DrawableRes int drawable) {
+    Resources resources = context.getResources();
+    if (Versions.isLollipop()) {
+      return resources.getDrawable(drawable, context.getTheme());
```

Comment:
I'm not sure if compat classes APIs change a lot, but I guess we can use them directly and if APIs brake we can think about wrapping them in our classes.